src/components/internal/InternalInput.tsx

```diff
@@ -127,6 +129,7 @@ const InternalInput = (props: InternalInputProps) => {
     >
       <input
         css={inputCSS}
+        aria-label={ariaLabel}
```

Comment:
why do we need this?

Reply:
Required to allow better accessibility in the `DateInput`, allowing us to add labels to `day`, `month` and `year` fields without requiring us to wrap it in a field. see: https://github.com/moroshko/basis/pull/233/files#diff-63dfb781167288da6af3955abb7691adddeb55c1a059c713870b5939657e3468R211

it also makes testing much easier as we can fire events directly on each field using the label see:
https://github.com/moroshko/basis/pull/233/files#diff-f4f0b4b1da9a2870c586a00aeb288a7b61d3a32fbc1dd710bb5c9497db92f1e1R255